### ISSUE TITLE
Auto android bugs

### ISSUE THREAD

**Issue by 22marco77** [maxrave-dev/SimpMusic#317]
On the phone the application works great, a few crashes but nothing serious. Except that when I try to open the app from Android Auto it won't open I see black screen.
Telefono Huawei mate 20 pro emui 12 Android 10 ultima versione Android auto.

**Comment by owencz1998** [comment]
Can you send the log when it next crashes maybe it can be apart of the issue.

Thanks.

**Comment by 22marco77** [comment]
> Can you send the log when it next crashes maybe it can be apart of the issue.
> 
> Thanks.

Of course, if you tell me where to recover it and how to create the log register relating to the problem. Thank you

**Comment by owencz1998** [comment]
When it crashes it should show dialog with copy log something like that unless it's crashing without showing the message

**Comment by 22marco77** [comment]
> When it crashes it should show dialog with copy log something like that unless it's crashing without showing the message 

Ok, in fact I published the crash in phone mode, in android auto I didn't see any dialog box evidently in android auto it doesn't display

**Comment by 22marco77** [comment]
![Uploading IMG_20240318_071105.jpg …]()

It doesn't load the Downloaded music and I don't see the online part to listen to the one Proposed by Spotify or whatever.

**Comment by 22marco77** [comment]
![Uploading IMG_20240318_071105.jpg …]()

It doesn't load the Downloaded music and I don't see the online part to listen to the one Proposed by Spotify or whatever.

**Comment by maxrave-dev** [comment]
Your image didnt be uploaded

**Comment by maxrave-dev** [comment]
Does all content worked on your phone?

**Comment by 22marco77** [comment]
> Does all content worked on your phone?

![313769776-8556cee8-bc2c-41d9-8343-00e675a0698d](https://github.com/maxrave-dev/SimpMusic/assets/24650739/387fb5d1-f0c8-47c8-ab41-cc40cb98e179)
![313769789-3e0e6f3a-8ecd-4199-bba2-beed543c4e18](https://github.com/maxrave-dev/SimpMusic/assets/24650739/38444fe4-f0c4-4a84-b461-7c426c5cadff)

contents work in my phone.
in Android Auto I notice a great slowness in reading data, with very long waiting times that leave the screen black as in the photo posted previously. Would it be possible to have better integration of the application into Android Auto? Thank's for your job.

**Comment by CrazyWolf13** [comment]
This has been resolved with 0.2.19
